lib/pyszn/injection.py

```diff
@@ -155,7 +157,7 @@ def parse_attribute_injection(
 
                 # Nodes
                 if 'nodes' in modifier:
-                    for node in _expand_nodes(filename, modifier['nodes']):
+                    for node in _expand_nodes(filename, modifier['nodes'], base_szn_dir=base_szn_dir):
```

Comment:
there is a corollary on what carlos said earlier: if the method defines the argument as positional, it should be passed as possitional (which is the case of expand_* methods here)